tests/e2e/fixtures/cairo1_project/libraries/external_lib_bar/bar.cairo

```diff
@@ -0,0 +1,3 @@
+fn bar() {
+    external_lib_foo::foo::foo();
+}
```

Comment:
All these files for external libs are just copied from `tests/integration/cairo1_compiler_bindings/data`, isn't there a way we could reuse already-existent files?

Reply:
I am not really a fan of over anti-duplicating. IMO we should keep stuff from e2e tests separated. This code is tightly coupled to the tests and abstracting it make it much harder to modify and adjust.

Reply:
Sure, it was just a thought, fair enough.